contrib/julienschmidt/httprouter/option.go

```diff
@@ -15,3 +20,10 @@ func WithServiceName(name string) RouterOption {
 		cfg.serviceName = name
 	}
 }
+
+// WithSpanOptions applies the given set of options to the spans started by the router.
```

Comment:
Can we make this singular? s/spans/span

Reply:
This takes in a slice of span options so technically singular isn't correct as I would like the ability to add multiple span options when initializing the router.